solidity_contracts/src/PlainOpcodes/Counter.sol

```diff
@@ -17,17 +17,21 @@ contract Counter {
         count += 1;
     }
 
-    function decUnchecked() public {
-        unchecked {
-            count -= 1;
-        }
+    function dec() public greaterThanZero {
+        count -= 1;
     }
 
     function decInPlace() public greaterThanZero {
         count--;
     }
 
-    function dec() public greaterThanZero {
+    function decUnchecked() public {
+        unchecked {
+            count -= 1;
+        }
+    }
+
+    function decUnsafe() public {
```

Comment:
Idk if unsafe is the right name for it, since it's behaving exaclty like "safemath.sol"